README.md

```diff
@@ -1133,6 +1133,8 @@ PROJECT(myproject)
 
 # fetch latest argparse
 include(FetchContent)
+set(ARGPARSE_BUILD_TESTS OFF CACHE INTERNAL "Turn off building argparse tests")

```

Comment:
These options should be `OFF` by default for subprojects (i.e. `NOT PROJECT_IS_TOP_LEVEL`)

Reply:
I honestly agree with you. The only thing is that PROJECT_IS_TOP_LEVEL is [available in CMake versions 3.21+](https://cmake.org/cmake/help/latest/variable/PROJECT_IS_TOP_LEVEL.html) and argparse uses CMake minimum version 3.12.4 as of now. There are ways around this of course like checking if PROJECT_NAME is defined ([Catch2 uses this tactic](https://github.com/catchorg/Catch2/blob/devel/CMakeLists.txt)) which I plan to use. The PR should be up in   around the next hour if everything works out. Also, [the install targets shouldn't be set when using as a subproject](https://github.com/catchorg/Catch2/issues/1373) too, and as of now ARGPARSE_INSTALL defaults to ON.